src/test/cpp/test/CmdReader.cpp

```diff
@@ -0,0 +1,137 @@
+#include "pch.h"
+#include "winrt/Windows.Foundation.h"
+#include "cmd_reader.h"
+#include <fstream>
+
+using namespace xlang::cmd;
+
+TEST_CASE("CmdReader")
```

Comment:
@kennykerr why are test names PascalCased instead of snake_cased?

Reply:
Originally so that I could easily distinguish between library types and test code, but that's not a compelling reason. I can switch it to make it more consistent overall.

Reply:
I actually think that's a compelling reason and switching it seems like a bunch of busy work when we have more important things to do